tasks/deployment_manager/task.ts

```diff
@@ -152,5 +153,9 @@ task('migrate', 'Runs migration')
       }
 
       await runMigration(dm, prepare, enact, migration, overwrite);
+
+      if (!simulate && enact) {
```

Comment:
Does this depend on `overwrite` at all?

Reply:
Should it? It seems like `overwrite` is only for controlling writing the migration artifacts, so I'm hesitant to repurpose it for this as well. WDYT about introducing a new flag for writing `enacted` to the migration script? It does seem like we are getting a bit crowded with all the different flags though.

Reply:
🤔  `!noEnacted && enact` might be nice

Reply:
I added a `setEnacted` flag that is always set in the `Enact Migration` task.

Reply:
Changed this flag to `noEnacted`